src/features/helper/configuration.ts

```diff
@@ -322,14 +322,17 @@ export default class Configuration {
       // - the path of the folder opened in VS Code
       workspaceFolder: rootPath,
 
+      // - the current opened file's workspace folder
+      fileWorkspaceFolder: (editor) ? vscode.workspace.getWorkspaceFolder(editor?.document.uri)?.uri.fsPath : undefined,
```

Comment:
Note: this replicates what is natively done in VS Code: https://github.com/microsoft/vscode/commit/876d1f09cc242f00ffcbec9766d13ce9503f90a1#diff-1f2c612d13bcf40cd390238ba712e009183a44f113a48da390f7910d2834a5feR964